### PR TITLE
Signing providers samples: Add encodeURIComponent().

### PR DESCRIPTION
**Pull request by andreibancioiu**
#### Description of the pull request (what is new / what has changed)

Improved samples for signing providers (erdjs).

#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [ ] update documentation that is not relevant anymore
- [x] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/erdjs/erdjs-signing-providers.md [hunk 1]
```diff
@@ -43,7 +43,8 @@ The following provider URLs [are defined](https://github.com/ElrondNetwork/elron
 Then, ask the user to log in:
 
 ```
-await provider.login({ callbackUrl: "http://my-dapp" });
+const callbackUrl = encodeURIComponent("http://my-dapp");
+await provider.login({ callbackUrl: callbackUrl });
```
Comment (arhtudormorar) [inline docs/sdk-and-tools/erdjs/erdjs-signing-providers.md:47]: could be just `await provider.login({ callbackUrl });`

File: docs/sdk-and-tools/erdjs/erdjs-signing-providers.md [hunk 2]
```diff
@@ -72,8 +73,7 @@ const authToken = "aaaabbbbaaaabbbb";
 
 // A server-side handler used to acknowledge, validate and honour
 // the relationship between "authToken" and the Elrond address of the user
-const callbackUrl = "https://my-dapp/on-wallet-login"
-
+const callbackUrl = encodeURIComponent("https://my-dapp/on-wallet-login");
 await provider.login({ callbackUrl: callbackUrl, token: authToken });
```
Comment (arhtudormorar) [inline docs/sdk-and-tools/erdjs/erdjs-signing-providers.md:77]: `await provider.login({ callbackUrl, token: authToken });
`

Comment (andreibancioiu) [reply docs/sdk-and-tools/erdjs/erdjs-signing-providers.md]: Fixed.

File: docs/sdk-and-tools/erdjs/erdjs-signing-providers.md (hunk 1 above)
Comment (andreibancioiu) [reply docs/sdk-and-tools/erdjs/erdjs-signing-providers.md]: Fixed.

**Review by arhtudormorar — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**